pkg/storage/storage_get.go

```diff
@@ -108,7 +112,16 @@ func (s *Storage) Get(ctx context.Context, gi *GetInput) (*GetOutput, error) {
 			aggregationType = averageAggregationType
 		}
 
+		timelineKey := "*"
```

Comment:
What does a `*` group mean?

Reply:
It's for the case when there's no value for a certain key:value pair. E.g if somebody ingested data at `app{foo="bar"}`, and you're grouping by `baz`. in this example `baz` is empty, so the data written to `app{foo="bar"}` becomes `*`. Data written to `app{baz="bazz"}` becomes `bazz`.

Ideally we should come up with something better than `*` I think.